src/graphics/bargraphics.jl

```diff
@@ -1,3 +1,5 @@
+if VERSION < v"0.6-"
```

Comment:
please be more specific with VERSION cutoffs like this, `where` isn't going to work for all 0.6-dev versions